src/components/__fixtures__/helpers.js

```diff
@@ -27,3 +28,12 @@ export function mountFromProps (propsData = {}) {
 
   return vm
 }
+
+export function coreHasFeature (feature) {
+  if (feature === REFERENCE_ICON_BY_STYLE || feature === ICON_ALIASES) {
+    return parse.icon
+  }
+}
+
+export const REFERENCE_ICON_BY_STYLE = 0x00
+export const ICON_ALIASES = true
```

Comment:
@jasonlundien let's stick with the hex notation we've begun.
```suggestion
export const ICON_ALIASES = 0x01
```

Reply:
fixed it up !